site/src/examples/card/BackgroundImage.tsx

```diff
@@ -0,0 +1,30 @@
+import { Card, H3, Link, StackLayout, Text } from "@salt-ds/core";
+import type { CSSProperties, ReactElement } from "react";
+
+export const BackgroundImage = (): ReactElement => {
+  return (
+    <Card
+      style={
+        {
+          backgroundImage: 'url("/img/examples/backgroundImg.png")',
+          backgroundSize: "cover",
+          //   Make sure text are visible on both light and dark modes
```

Comment:
Change "Make sure text are visible on both light and dark modes" to "Make sure text meets color contrast standards in both light and dark modes."

Reply:
updated together with content-bold. Did use inherit, given it's not how user would generally implement